### ISSUE TITLE
Use template strings for string interpolation

### ISSUE THREAD

**Issue by tofumatt** [localForage/localForage#740]
We should start using ES6's much nicer string interpolation instead of the string concatenation we currently do. Bonus points: we can use stuff like `oneLine` from the [common-tags](https://www.npmjs.com/package/common-tags) package like we do on the Add-ons repos.

**Comment by tricinel** [comment]
Still need help with this? Would love to tackle it...

**Comment by thgreasi** [comment]
👍

**Comment by tricinel** [comment]
Awesome. Will get a PR in this week.

**Comment by tricinel** [comment]
Right, so I've gone through the src files and changed to template strings. I thought they'd be more, but there weren't that many instances actually.

Having gone through the Contributing.md file, there are some mentions of .jshint and .jscs - but I noticed you switched to eslint.

Also, while running npm test...Everything passes until I get to the actual mocha tests, where it gets stuck...And opening the browser with that connect web server that was started gives me errors in the console - although I should mention the exact same errors are with a clean master.

So...I'm a bit stuck on this one... :)

**Comment by thgreasi** [comment]
Feel free to open the PR an we can elaborate on it.

**Comment by tricinel** [comment]
Alright, cool. I just did...willing to make it work properly ;)

**Comment by savalaram-redkar** [comment]
I'm interested in working on the issue. Before I start, I wanted to check if the issue is still relevant?
